kore/test/Test/Kore/BugReport.hs

```diff
@@ -82,6 +90,40 @@ test_parse =
             actual = expect >>= parseKoreLogOpts . unparseKoreLogOptions
         getParseResult expect === getParseResult actual
 
+test_bugReportOption :: TestTree
+test_bugReportOption =
+    testGroup
+        "Parse BugReportOption"
+        [ testCase "Parse BugReportEnable" enableAssert
+        , testCase "Parse BugReportDisable" disableAssert
+        , testCase "Parse BugReportOnError" onErrorAssert
+        ]
+  where
+    enableAssert =
+        assertParse
+            ["--bug-report", "fileName"]
+            (BugReportEnable $ BugReport "fileName")
+
+    disableAssert =
+        assertParse
+            ["--no-bug-report"]
+            BugReportDisable
+
+    onErrorAssert =
+        assertParse
+            []
+            BugReportOnError
+
+    assertParse :: [String] -> BugReportOption -> Assertion
```

Comment:
Using `HasCallStack` will ensure that failing tests are attributed to the right line number:

```suggestion
    assertParse :: HasCallStack => [String] -> BugReportOption -> Assertion
```